dp3t-sdk/sdk/src/main/java/org/dpppt/android/sdk/internal/SyncWorker.java

```diff
@@ -134,9 +135,12 @@ public void doSync() throws Exception {
 
 				try {
 					uploadPendingKeys(context);
-					doSyncInternal(context);
-					Logger.i(TAG, "synced");
-					AppConfigManager.getInstance(context).setLastSyncNetworkSuccess(true);
+
+					if (DP3T.isTracingEnabled(context) && Boolean.TRUE.equals(GaenStateCache.isGaenEnabled())) {
```

Comment:
If the app's process has been just started (to execute the worker), `GaenStateCache.isGaenEnabled()` might still return null, as its first invalidation might not have finished yet.